src/datastore.py

```diff
@@ -1,68 +1,43 @@
 from pyspark.sql import DataFrame, DataFrameWriter, DataFrameReader
-from pyspark.sql.types import *
+from pyspark.sql.functions import lit
+from schema import schema_csv, schema_meta
 
 
 class DataStore:
-    schema_csv = StructType([
-        StructField('Date', DateType()),
-        StructField('Open', DoubleType()),
-        StructField('High', DoubleType()),
-        StructField('Low', DoubleType()),
-        StructField('Close', DoubleType()),
-        StructField('Adj Close', DoubleType()),
-        StructField('Volume', LongType())
-    ])
 
-    schema_meta = StructType([
-        StructField("Nasdaq Traded", StringType()),
-        StructField("Symbol", StringType()),
-        StructField("Security Name", StringType()),
-        StructField("listing Exchange", StringType()),
-        StructField("Market Category", StringType()),
-        StructField("ETF", StringType()),
-        StructField("Round Lot Size", DoubleType()),
-        StructField("Test Issue", StringType()),
-        StructField("Financial Status", StringType()),
-        StructField("CQS Symbol", StringType()),
-        StructField("NASDAQ Symbol", StringType()),
-        StructField("NextShares", StringType())
-    ])
-
-    def __init__(self, ses, config=None):
-        self.spark = ses
+    def __init__(self, spark, config=None):
+        self.spark = spark
         self.config = config
 
+    def rename_columns(self, df: DataFrame):
+        dic = {}
+        for col in df.columns:
+            lis = col.split()
+            new_name = lis[0].lower()
+            if len(lis) > 1:
+                new_name = f"{lis[0].lower()}{lis[1].capitalize()}"
+            dic[col] = new_name
+
+        return df.withColumnsRenamed(dic)
+
     def load_symbol(self, symbol) -> DataFrame:
-        df_meta = self.load_metadata()
-        df_symbol = self.load_symbol_raw(symbol)
-        df_meta.join(df_symbol)  # enrich symbol with long symbol name
+        df_meta = self.load_metadata().select('symbol', 'securityName')
+        df_sym = self.load_symbol_raw(symbol).withColumn('sym', lit(symbol))
```

Comment:
It would be consistent if we used the name `symbol` here as well. In the result DF should be only one single column named `symbol`.